frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/Toolbar/ZoomControls/ZoomControls.tsx

```diff
@@ -7,14 +9,25 @@ import styles from './ZoomControls.module.css'
 export const ZoomControls: FC = () => {
   const zoomLevel = useStore((store) => store.transform[2])
   const { zoomIn, zoomOut } = useReactFlow()
+  const { showMode } = useUserEditingStore()
 
   const handleClickZoomOut = useCallback(() => {
+    toolbarActionLogEvent({
+      element: 'zoom',
+      zoomLevel: zoomLevel.toFixed(2),
+      showMode: showMode,
+    })
     zoomOut()
-  }, [zoomOut])
+  }, [zoomOut, zoomLevel, showMode])
 
   const handleClickZoomIn = useCallback(() => {
+    toolbarActionLogEvent({
+      element: 'zoom',
+      zoomLevel: zoomLevel.toFixed(2),
+      showMode: showMode,
```

Comment:
[nits]

```suggestion
      showMode,
```

Reply:
Thank you🙏!
I've addressed all three of the points raised.↑
Fixed: https://github.com/liam-hq/liam/pull/326/commits/932ce8233165fb04e30405379d08ba7d8530eded